src/public/app/widgets/buttons/calendar.js

```diff
@@ -123,7 +123,6 @@ export default class CalendarWidget extends RightDropdownButtonWidget {
 
             if (note) {
                 appContext.tabManager.getActiveContext().setNote(note.noteId);
-                this.hideDropdown();
```

Comment:
Seems the calendar user experience has been degraded:

- Clicking on a day will no longer dismiss the calendar.
- All items appear underlined, whereas in the past only the ones that have a corresponding note do that.